index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portfolio by Abenezer Samuel</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="toolbar">
+    <nav class="nav-items">
+      <h2 class="logo">Ebenezer</h2>
+      <img src="./image/menu.png" alt="menu bar image" />
+    </nav>
+  </header>
+  <section class="bg">
+    <section class="headline">
+    <h1 class="Primay-text">
```

Comment:
- [ ] please set your background-size to cover, so the headline background image can be responsive.
![Screenshot (1211)](https://github.com/abenisamuel11/mobile-version-finish-new-season2/assets/65298048/9e0b2e66-1552-42d4-95ac-1804111e93e7)